bugRzilla_review.Rmd

```diff
@@ -50,44 +49,24 @@ To install SQL on Ubuntu one can refer a blog post by [digitalocean](https://www
 To install MySQL workbench on Ubuntu one can refer a blog post by [linuxhint](https://linuxhint.com/installing_mysql_workbench_ubuntu/)
 </font>
 
-## Download R_bugzilla data
+## Download R bugzilla data
 
 <font size ="4">
 <ol>
 <li>
-The R_bugzilla data can be downloaded from  [link](https://bugs.r-project.org/db/R-bugs.sql.xz).
+The R Core have made a dump of the R Bugzilla database available for analysis can be downloaded from [link](https://bugs.r-project.org/db/R-bugs.sql.xz).
```

Comment:
Great! it would be even better if you mention the date of the dump: 25/03/2021

Reply:
sure!